### PR TITLE
AudioEngine: Include LFE mixing setting

### PR DESCRIPTION
**Pull request by fritsch**
Follow up of: #18495

After the exiv changes ubuntu is broken again. One time too much in two days ;-)  - so jenkins can build for me again.

### CODE REVIEW

**Comment by fritsch** [comment]
2024-06-27 20:05:26.654 T:1608438   debug <general>: ffmpeg[0x5f4ae1df2140]: [SWR] Using fltp internally between filters
2024-06-27 20:05:26.654 T:1608438   debug <general>: ffmpeg[0x5f4ae1df2140]: [SWR] Matrix coefficients:
2024-06-27 20:05:26.654 T:1608438   debug <general>: ffmpeg[0x5f4ae1df2140]: [SWR] FL: FL:1.000000 FR:0.000000 FC:0.707107 LFE:0.707107 SL:0.707107 SR:0.000000
2024-06-27 20:05:26.654 T:1608438   debug <general>: ffmpeg[0x5f4ae1df2140]: [SWR] FR: FL:0.000000 FR:1.000000 FC:0.707107 LFE:0.707107 SL:0.000000 SR:0.707107

**Comment by fritsch** [comment]
jenkins build this please (give me the next failures ;-))

**Comment by HiassofT** [comment]
Thanks a lot, I tested this change on Omega and Piers the last couple of days, and it seems to be working just fine!

As the patch cleanly picks into the Omega branch I also wouldn't mind if you'd do a backport PR :)

**Comment by fritsch** [comment]
@garbear here is the change.

If it fits needs and no blocking points found, feel free to go ahead.

File: xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp [hunk 1]
```diff
@@ -2666,6 +2669,7 @@ void CActiveAE::LoadSettings()
   m_settings.atempoThreshold = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_ATEMPOTHRESHOLD) / 100.0;
   m_settings.streamNoise = settings->GetBool(CSettings::SETTING_AUDIOOUTPUT_STREAMNOISE);
   m_settings.silenceTimeoutMinutes = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_STREAMSILENCE);
+  m_settings.mixSubLevel = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_MIXSUBLEVEL) / 100.0;
```
Comment (fritsch) [inline xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp:2672]: I don't clamp here for a reason ... uses with the 200% bass need ... can just edit their scan and shoot in their foot :-)

Comment (TheTyrius) [reply xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp:2672]: Sorry if this is obvious, by 
> can just edit their scan

do you mean manually adjusting the settings.xml?

In the past week I have finally watched a few movies with `mixsublevel` and depending on genre a setting of `150-200` worked very well (and is usually less "overkill" than the number would indicate, when the LFE is often stored at -10dB). Thanks again for the patch! I've already had a lot of fun with the preserved bass-effects in this build.

Comment (fritsch) [reply xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp:2672]: Yeah - that's what I meant. I don't want to make it harder for the people with that need. While I want to stick to the default sqrt(2) values - especially when you make two speakers from one sub.

**Comment by fritsch** [comment]
Will merge in upcoming days, else it might stand for another 2+ years ;-)

**Comment by rsalvaterra** [comment]
> Will merge in upcoming days, else it might stand for another 2+ years ;-)

Please do! Us folks with traditional stereo hi-fi setups have been getting nerfed downmixes since forever. :slightly_smiling_face:

**Comment by fritsch** [comment]
Good. Rebased a final time. Will push when green.

**Comment by fritsch** [comment]
jenkins build this please (seems arm64 has some temporary outtage and the webos guy is broken since longer).

**Comment by fritsch** [comment]
No idea what to do about those jenkins issues - does not seem related to this build.

**Comment by fritsch** [comment]
jenkins build this please - maybe one more build gets green this time. (Final try).

**Comment by fritsch** [comment]
jenkins build this please

**Pull request merged**